addons/sourcemod/scripting/n_arms_fix.sp

```diff
@@ -441,7 +441,7 @@ public void PrecacheModels(bool map)
     char modelsPath[64];
     Format(modelsPath, sizeof(modelsPath), "%s", g_bLegacyModels ? LEGACY_MODELS_PATH : MODELS_PATH);
 
-    if (map) {
+    if (map && HasMapModels(NTEAM_T) && HasMapModels(NTEAM_CT)) {
```

Comment:
error 017: undefined symbol "NTEAM_T" / "NTEAM_CT", you probably mean TEAM_T / TEAM_CT?

Reply:
Yes. Sorry for that. I re-defined that in my local file.